# PR pending: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66001

diff --git a/node_modules/@types/lodash-es/property.d.ts b/node_modules/@types/lodash-es/property.d.ts
index 05317ae..0165866 100755
--- a/node_modules/@types/lodash-es/property.d.ts
+++ b/node_modules/@types/lodash-es/property.d.ts
@@ -1,2 +1,4 @@
-import { property } from "lodash";
+import { property, PropertyPath } from "lodash";
+
+export { PropertyPath };
 export default property;
