# PR merged: https://github.com/cssinjs/jss/pull/1637

diff --git a/node_modules/jss/dist/jss.esm.js b/node_modules/jss/dist/jss.esm.js
index 915ecea..5c71e64 100644
--- a/node_modules/jss/dist/jss.esm.js
+++ b/node_modules/jss/dist/jss.esm.js
@@ -456,7 +456,7 @@ function () {
 
   return ConditionalRule;
 }();
-var keyRegExp = /@media|@supports\s+/;
+var keyRegExp = /@container|@media|@supports\s+/;
 var pluginConditionalRule = {
   onCreateRule: function onCreateRule(key, styles, options) {
     return keyRegExp.test(key) ? new ConditionalRule(key, styles, options) : null;
