diff --git a/node_modules/react-timeago/es6/index.js b/node_modules/react-timeago/es6/index.js
index c97a2f7..48912c0 100644
--- a/node_modules/react-timeago/es6/index.js
+++ b/node_modules/react-timeago/es6/index.js
@@ -50,6 +50,9 @@ export default function TimeAgo({
       }
     };
   }, [date, live, maxPeriod, minPeriod, now, timeNow]);
+  useEffect(() => {
+    setTimeNow(now());
+  }, [date]);
   const Komponent = component;
   const then = dateParser(date).valueOf();
   if (!then) {
diff --git a/node_modules/react-timeago/lib/index.js b/node_modules/react-timeago/lib/index.js
index 28d643c..d9934ed 100644
--- a/node_modules/react-timeago/lib/index.js
+++ b/node_modules/react-timeago/lib/index.js
@@ -83,6 +81,9 @@ function TimeAgo(_ref) {
       }
     };
   }, [date, live, maxPeriod, minPeriod, now, timeNow]);
+  (0, React.useEffect)(function () {
+    setTimeNow(now());
+  }, [date]);
   var Komponent = component;
   var then = (0, _dateParser["default"])(date).valueOf();
   if (!then) {
